python/tank/pipelineconfig.py

```diff
@@ -358,7 +358,7 @@ def _get_pipeline_config_file_location(self):
             constants.PIPELINECONFIG_FILE
         )
 
-    def _get_yaml_cache_location(self):
+    def get_yaml_cache_location(self):
```

Comment:
As I mentionned above, this is now a public method on an internal API, so it's okay to expose and makes some tests cleaner since we can simply rely on the computed version.